//Sin Try Catch
console.log(5 + 4);

hola(); //Función no definida

console.log("Hola mundo");

//Usando Try Catch
console.log(5 + 4);
try {
    hola(); //Función no definida
}catch(error){
    console.log("Error");
}

console.log("Hola mundo");